src/apps/Agreement/DisputableApps/DisputableApps.js

```diff
@@ -0,0 +1,221 @@
+import React from 'react'
+import {
+  DataView,
+  ContextMenu,
+  ContextMenuItem,
+  useTheme,
+  GU,
+  useLayout,
+  Help,
+  TokenBadge,
+} from '@aragon/ui'
+import { KnownAppBadge } from '../../../templates/kit'
+import InfoField from './../InfoField'
+import { AppItemsType } from './../prop-types'
+import SubtleLabel from './SubtleLabel'
+import TokenAmount from './TokenAmount'
+
+const EXPANDABLE_ROW_GAP = `${3 * GU}px`
+
+const DisputableApps = React.memo(({ items }) => {
+  const { layoutName } = useLayout()
+  const theme = useTheme()
+
+  return (
+    <DataView
+      fields={[
+        { label: 'App', priority: 1, childStart: true },
+        { label: 'Actions', align: 'left' },
+      ]}
+      entries={items}
+      renderEntry={entry => renderEntry(entry, layoutName)}
+      renderEntryExpansion={entry => renderEntryExpansion(entry, layoutName)}
+      renderEntryActions={entry => renderEntryActions(entry, theme)}
+    />
+  )
+})
+
+function renderEntry({ allowedActions }, layoutName) {
+  return [
+    <div
+      css={`
+        display flex;
+        align-items: center;
+
+        /* Height must match expansion button to align nicely */
+        ${(layoutName === 'medium' || layoutName === 'large') &&
+          'height: 32px;'}
+        
+      `}
+    >
+      <KnownAppBadge appName="voting.aragonpm.eth" label="Voting" />
```

Comment:
This is likely the wrong badge to use here, but it was the easiest to mock out before getting the data in.

Reply:
Thought so! I'll make this change when we get the data in.